src/js/errorboundary.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
```

Comment:
Good stuff!

Reply:
Over time this should allow catching & handling errors a bit closer to the erroneous area - once they are placed a bit further around